<launch>
    <!--<include file="$(find pr2_machine)/$(env ROBOT).machine" />-->
    <!-- for actual production run rsync w/ c2 and add machine="c2" to tag below -->
    <node pkg="marker_detect" name="marker_detector_stream_basket" type="marker_detector_node" output="screen" args="output_x">
    	<param name="num_markers" type="int" value="100" />
    	<param name="camera_frame" type="string" value="narrow_stereo_optical_frame" />
    	<param name="target_frame" type="string" value="base_footprint" />
    	<param name="scale_factor" type="double" value="0.027" />
    	<param name="no_x_output" type="bool" value="false" />
        <param name="frame_rate" type="int" value="10" />
        <!-- proportional to size of markers -->
        <param name="block_size" type="int" value="31" />
        <!-- increase if poor contrast -->
        <param name="offset" type="double" value="15.0" />
    	<remap from="image" to="/narrow_stereo/left/image_rect" />
        <remap from="marker_detector" to="marker_detector_basket"/>
    </node>
</launch>
